.github/workflows/publish.yml

```diff
@@ -6,5 +6,18 @@ jobs:
   publish:
     runs-on: ubuntu-latest
     steps:
-      - name: Publish dummy job
-        run: echo "Hello publish dummy job!"
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Node
+        uses: ./.github/actions/node
+
+      - name: Publish
+        env:
+          SLACK_TOKEN: ${{ secrets.SLACK_TOKEN }}
+          SLACK_API_POST_RELEASE_MESSAGE: ${{ secrets.SLACK_API_POST_RELEASE_MESSAGE }}
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" >> .npmrc
+          echo "hello publish job!"
+          yarn publish-packages
```

Comment:
Just for safety reasons, since we keeping `publish.mjs` file, I suggest using here in action directly `lerna publish` command as it will be needed to run it with `--yes` flag to have autoprompt on a question about the release, which we do not want to have in publish.mjs file if we would like to keep it :)

Reply:
can be abstracted from the publish mjs and included here as "zx postChangelogToSlack" smth like that :)

Reply:
it's still not posting correctly the version of the package, so maybe you can fix also this 🤔